scripts/devnet-curio/lotus.dockerfile

```diff
@@ -1,5 +1,5 @@
 # Lotus binaries image, to be used in the local devnet with Forest.
-FROM golang:1.21-bookworm AS lotus-builder
```

Comment:
The 2 lotus.dockerfile(s) are identical, should we delete this one?

Reply:
I think we should re-use the existing one. That said, @lemmih believes both composes will eventually get merged (or rather, the lotus-miner will get replaced by curio). See comments in https://github.com/ChainSafe/forest/pull/4760